Makefile

```diff
@@ -12,7 +12,8 @@ bump-minor:
 	bumpversion minor
 
 deploy-pypi: clear
-	python setup.py sdist bdist_wheel
+	python3 -c "import sys; sys.version_info >= (3, 5, 3) or sys.stdout.write('Python version must be greatest then 3.5.2\n') or exit(1)"
```

Comment:
Shouldn't this be `sys.stdout.write('Python version must be greatest then 3.5.3\n') and exit(1)`?

Reply:
No, Python evaluates boolean conditions lazily.
Result `sys.stdout.write('Foo')` is `None`, `bool(None) == False`.
In this line conditions will be interpreted as `False or False or <action>`.
If last operator will be changed from `or` to `and` then `<action>` not be called.

P.S. Sorry, for my English😅

Reply:
Ah, I wasn’t even thinking about the return type of the write. I was thinking of success/failure. I hadn’t had my coffee yet :)